.github/workflows/release.yml

```diff
@@ -9,12 +9,12 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
       - name: Setup Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v4
         with:
-          go-version: 1.16
+          go-version: '1.21.3'
```

Comment:
This question applies to all the GHA workflows:

Do we need to be that explicit about the version? I would say stating `1.21` would be enough, regardless of the _patch_ version. WDYT?